### ISSUE TITLE
Lang File Bug

### ISSUE THREAD

**Issue by Answordy** [SuperMartijn642/ScarecrowsTerritory#13]
Mod displays itemGroup.scarecrowsterritory in creative inventory instead Scarecrow's Territory. Seems to be a lang file issue.
"scarecrowsterritory.itemGroup": "Scarecrows' Territory" should be "itemGroup.scarecrowsterritory": "Scarecrows' Territory"

**Comment by SuperMartijn642** [comment]
What version of Minecraft and Scarecrows' Territory are you using?

**Comment by Answordy** [comment]
The latest, 1.1.4

**Comment by SuperMartijn642** [comment]
And what Minecraft version are you using?

**Comment by Answordy** [comment]
1.18.2, but should work the same on any version, considering there wasn't any structural change (as far as I know) in lang files between the versions supported by Scerecrow's Territory 1.1.4, where the creative inventory tab was added. So I found these infos not relevant, thats why I forget to provide them in the first place, sorry for that. I will also give you the screenshots, so there won't be any infos missing.

The issue:
![2022-04-28_15 46 16](https://user-images.githubusercontent.com/82783515/165766891-65653217-f325-4157-882f-870b11d3674c.jpg)

Fast fix (as mentioned above) with resourcepack:
![2022-04-28_15 48 13](https://user-images.githubusercontent.com/82783515/165767071-513fc990-de86-47c1-9dc9-1a82c81f429f.png)
